xhr.bs

```diff
@@ -957,7 +957,7 @@ method must run these steps:
         <a>state</a> changes.
 
        <li><p><a>Fire a progress event</a> named
-       <a event><code>progress</code></a> with <var>response</var>'s
+       <a event><code>progress</code></a> at [=this=] with <var>response</var>'s
```

Comment:
The intended target isn't clear in context. Firefox, Chrome, Edge, and Safari all fire this event at the request object.